target/linux/ath79/dts/arxxxx_fortinet_loader.dtsi

```diff
@@ -40,6 +40,21 @@
 				label = "u-boot";
 				reg = <0x000000 0x040000>;
 				read-only;
+				compatible = "nvmem-cells";
```

Comment:
space after read-only.

Reply:
actually nvm. nvmem-cells is not needed anymore.

Reply:
So I can drop lines 43-45?

Reply:
yeah. See https://github.com/neheb/openwrt/commit/a7bfe15a3b675259835dd5271e2886b8ff010ffd for examples.